static/js/components/subject_page/block.tsx

```diff
@@ -666,7 +666,8 @@ function renderTiles(
           </p>
         );
       case "PLACE_OVERVIEW":
-        return <PlaceOverviewTile key={id} place={place} />;
+        // TODO(gmechali): Switch to server-side redirection
+        return tiles.length == 1 ? <></> : <PlaceOverviewTile key={id} place={place} />;
```

Comment:
I'm seeing this message flash (for < 1s) 
![Screenshot 2024-09-05 at 3 46 20 PM](https://github.com/user-attachments/assets/ab87ee6b-10c8-4832-a1a4-aa271be27473)

when visiting: 
http://localhost:8080/explore/#q=United+states+of+america&client=ui_query

Reply:
hmm interesting, im not seeing this at all. 
https://screencast.googleplex.com/cast/NjUyNDY1MzM0MTI0NTQ0MHw2ODkzN2MwMS01YQ

Reply:
I'm seeing it if i start from /explore : https://screencast.googleplex.com/cast/NDU0NzMwODc0OTMyNDI4OHwzZTI5NGYxMi1kMw

Reply:
So I wasnt able to see the first message you mentioned, but I did see the second one ("See relevant statistics based on the previous query."). That's now taken care of.